app/services/responsible_body_exporter.rb

```diff
@@ -6,30 +6,40 @@ class ResponsibleBodyExporter
 
   attr_reader :filename
 
-  def initialize(filename)
+  def initialize(filename = nil)
     @filename = filename
   end
 
   def export_responsible_bodies(query = responsible_bodies)
-    CSV.open(filename, 'w') do |csv|
-      csv << headings
-      query.find_each do |rb|
-        csv << [
-          rb.computacenter_identifier,
-          *build_name_fields_for(rb),
-          rb.address_1,
-          rb.address_2,
-          rb.address_3,
-          rb.town,
-          rb.postcode,
-        ]
+    if filename
```

Comment:
Will we still need the filename generation after this change?

Reply:
I guess it depends on how quickly CC transition to use list this and whether we need a fall back?

Reply:
yep fair enough, maybe worth scheduling a separate card for cleaning this up shortly after CC transition to the new process.